.circleci/config.yml

```diff
@@ -33,6 +33,12 @@ jobs:
           command: |
             . .venv/bin/activate
             pytest --cov=cms -n auto --create-db --nomigrations
+      - run:
+          name: pytest-1.11
+          command: |
+            . .venv/bin/activate
+            pip install 'django>=1.11,<1.12'
```

Comment:
Should this not be using 2.2?

Reply:
The other pytest block above it tests for 2.2 (which is automatically installed by setup.py). This is a second run of tests to ensure that it still works under 1.11.